helm_deploy/laa-estimate-eligibility/values/production.yaml

```diff
@@ -37,9 +37,6 @@ ingress:
   annotations:
     external-dns.alpha.kubernetes.io/set-identifier: "check-client-qualifies-laa-estimate-eligibility-laa-estimate-financial-eligibility-for-legal-aid-production-green"
     external-dns.alpha.kubernetes.io/aws-weight: "100"
-    nginx.ingress.kubernetes.io/enable-modsecurity: "true"
-    nginx.ingress.kubernetes.io/modsecurity-snippet: |
```

Comment:
question - is this the nginx stuff that was discussed in 'Dev time', the "stuff" being that nginx is the controller which handles the firewall in our ingress? 

I'm assuming we're removing this part as we no longer need modsecuirty to be our WAF.

Reply:
Exactly, we're removing the stuff that tells nginx "add modsecurity into the mix".